test/jellyfish_web/integration/server_notification_test.exs

```diff
@@ -412,13 +483,12 @@ defmodule JellyfishWeb.Integration.ServerNotificationTest do
     assert String.contains?(endpoint_id, "endpoint_id")
   end
 
-  def subscribe_on_notifications_and_connect_peer(conn) do
-    server_api_token = Application.fetch_env!(:jellyfish, :server_api_token)
+  defp subscribe_on_notifications_and_connect_peer(conn) do
     ws = create_and_authenticate()
-
     subscribe(ws, :server_notification)
 
-    {room_id, peer_id, peer_token, conn} = add_room_and_peer(conn, server_api_token)
+    {room_id, conn} = add_room(conn)
+    {peer_id, peer_token, conn} = add_peer(conn, room_id)
```

Comment:
Why did you make this change?

Reply:
So I could use the `add_peer` function in line `305`